report.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+echo "Enter cluster name (default my-cluster)"
+read cluster
+if [ -z $cluster ]; then
+	cluster="my-cluster"
+fi
+
+get_yamls() {
+	mkdir -p ./reports/"$1"s
+	resources=`oc get "$1"s  -l strimzi.io/cluster=$cluster | tail -n +2` # remove legend
```

Comment:
`oc get statefulset -l strimzi.io/cluster=my-cluster -o name` will just output the names, which might simplify this a bit.

Reply:
Oh, awesome! Thanks.

Reply:
I see one issue with this. For example `$ oc get deployment` returns `my-cluster-entity-operator` but the `-o name` option appends a prefix of resource type, which results in `deployments/my-cluster-entity-operator`. So we will need some postprocessing anyway :(

Reply:
`oc get deployments/my-cluster-entity-operator` works fine, so I don't think you need postprocessing.